packages/utils/package.json

```diff
@@ -19,7 +19,7 @@
     "@ethersproject/address": "5.6.0",
     "bignumber.js": "^9.1.0",
     "cids": "1.1.9",
-    "ethers": "5.6.9",
+    "ethers": "5.7.0",
```

Comment:
Are the other packages using this same version ?

Reply:
Yes, the `web` package was using `5.7.0` and this was creating some typescript issues for me locally.